### PR TITLE
Update console tab when kernel changes

### LINKED ISSUE #769: Console tab name doesn't update when you switch kernels
When you switch kernels in a Python 3 Console, the tab name does not update.
 
screen shot 2016-08-25 at 3 13 18 pm

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #769.

### CODE REVIEW

**Comment by afshin** [comment]
This changes the meaning of "Opened:" in the caption. Maybe it should be something like "Connected to kernel:" ... because it no longer indicates when the console itself was opened.

**Comment by blink1073** [comment]
Changed to "Connected" for brevity.

File: src/console/plugin.ts [hunk 1]
```diff
@@ -319,7 +319,7 @@ namespace Private {
       `Name: ${label}\n` +
       `Directory: ${ContentsManager.dirname(path)}\n` +
       `Kernel: ${displayName}\n` +
-      `Opened: ${dateTime(opened.toISOString())}`
+      `Connected: ${dateTime(opened.toISOString())}`
```
Comment (afshin) [inline src/console/plugin.ts]: 👍

Comment (afshin) [inline src/console/plugin.ts]: How about changed the variable name from `opened` as well, since it's got different semantics?

Comment (blink1073) [inline src/console/plugin.ts]: Done.

**Comment by afshin** [comment]
👍

**Pull request merged**